### PR TITLE
Use Google fonts and a fixed gradient background.

### PR DESCRIPTION
**Pull request by DaneWeber**
Is this an okay way of adding CSS style and of linking to an external asset (Google fonts) on all application pages?

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <title>MaintenanceApp</title>
+    <link rel="stylesheet"
```
Comment (jacohen10) [inline app/views/layouts/application.html.erb]: this should be fine, but normally try to use rails helpers when you can. example here: https://stackoverflow.com/questions/18508996/include-google-font-in-rails-app

File: app/assets/stylesheets/chores.scss [hunk 2]
```diff
@@ -1,3 +1,20 @@
 // Place all the styles related to the Chores controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+// Color scheme from https://www.behance.net/gallery/32154055/Minimalist-Color-Palettes-2015
+$a-yellow: #e5eec1;
```
Comment (jacohen10) [inline app/assets/stylesheets/chores.scss]: these variable names aren't super helpful. normally I'd $light-yellow, $dark-green, etc.

Comment (DaneWeber) [reply app/assets/stylesheets/chores.scss]: Do you have a recommendation for accomplishing the following: the five colors in pallette also have an order to them. I was hoping to embed the order in their names. Should I just leave that out of the names? Or make it more explicit with a name like `$first-yellow` or something similar?

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (DaneWeber) [reply app/views/layouts/application.html.erb]: I'll try later today, but I'm imagining I'd use the following line:

`<% stylesheet_link_tag "https://fonts.googleapis.com/css?family=Josefin+Sans|Roboto" %>`

Comment (jacohen10) [reply app/views/layouts/application.html.erb]: You're missing one argument. Double check the answers in stackoverflow

File: app/assets/stylesheets/chores.scss (hunk 2 above)
Comment (jacohen10) [reply app/assets/stylesheets/chores.scss]: Sounds strange to me, but I also don't understand how they're being used

**Review by jacohen10 — APPROVED** [review]

**Pull request merged**